js/constants/countries.js

```diff
@@ -0,0 +1,1000 @@
+const countries = [
```

Comment:
Yeah, so basically countryCodes.js is just an array of all the country codes? then append that to the array in app/locale?